packages/components/atoms/f-form-field/src/components/_tests/FormSelectionControl.test.js

```diff
@@ -0,0 +1,137 @@
+import { shallowMount } from '@vue/test-utils';
+import FormSelectionControl from '../FormSelectionControl.vue';
+
+describe('FormSelectionControl', () => {
+    let attrs;
+    let propsData;
+
+    beforeEach(() => {
+        attrs = {
+            disabled: null,
+            id: 'uniqueId1'
+        };
+
+        propsData = {
+            inputType: 'checkbox',
+            value: 'testValue'
+        };
+    });
+
+    it('should be defined', () => {
+        const wrapper = shallowMount(FormSelectionControl, { attrs });
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    describe('template ::', () => {
+        describe('value ::', () => {
+            it('should be set using the `value` prop', () => {
+                // Arrange & Act
+                const wrapper = shallowMount(FormSelectionControl, { propsData, attrs });
+
+                // Assert
+                expect(wrapper.find('input').element.value).toBe(propsData.value);
+            });
+        });
+    });
+
+    describe('props ::', () => {
+        describe('inputType ::', () => {
+            it.each([
+                [false, 'text'],
+                [true, 'checkbox'],
+                [true, 'radio']
+            ])('should return %s when value is %s', (expected, inputType) => {
+                // Arrange
+                const { validator } = FormSelectionControl.props.inputType;
+
+                // Act & Assert
+                expect(validator(inputType)).toBe(expected);
+            });
+        });
+    });
+
+    describe('computed ::', () => {
+        describe('isCheckbox ::', () => {
+            it('should return `true` when `inputType` is `checkbox`', async () => {
+                // Arrange
+                const wrapper = shallowMount(FormSelectionControl, { propsData, attrs });
+
+                // Act
+                const result = await wrapper.vm.isCheckbox;
+
+                // Assert
+                expect(result).toBe(true);
+            });
+
+            it('should return `false` when `inputType` is not `checkbox`', async () => {
+                // Arrange
+                propsData.inputType = 'radio';
+
+                const wrapper = shallowMount(FormSelectionControl, { propsData, attrs });
+
+                // Act
+                const result = await wrapper.vm.isCheckbox;
+
+                // Assert
+                expect(result).toBe(false);
+            });
+        });
+
+        describe('testId ::', () => {
+            describe('`attributes.name` is provided ::', () => {
+                it('should return an object using the `attribute.name`', async () => {
```

Comment:
Not _strictly_ the variable name so maybe better to just have it as plain text
```suggestion
                it('should return an object using the attribute name', async () => {
```